src/components/SFS/SFSFilters/SFSFilters.tsx

```diff
@@ -0,0 +1,178 @@
+import { useState } from 'react';
+
+import { SFSFilterProps } from './SFSFilters.types';
+
+import { getSFSFiltersUtilityClass } from './SFSFilters.classes';
+
+import { unstable_composeClasses as composeClasses } from '@mui/base';
+
+import { styled, useThemeProps } from '@mui/material/styles';
+import Button from '@mui/material/Button';
+import Drawer from '@mui/material/Drawer';
+import Typography from '@mui/material/Typography';
+
+import { IconClose, IconFilterVariantSmall } from '../../../icons';
+import { SFSButton } from '../SFSButton';
+
+type SFSFiltersOwnerState = {
+  classes?: SFSFilterProps['classes'];
+  isWithValue?: boolean;
+};
+
+const useUtilityClasses = (ownerState: SFSFiltersOwnerState) => {
+  const { classes } = ownerState;
+
+  const slots = {
+    root: ['root'],
+    filterButton: ['menuButton'],
+    closeButtonIcon: ['closeButtonIcon'],
+    menu: ['menu'],
+    menuHeader: ['menuHeader'],
+    menuFooter: ['menuFooter'],
+    resetButton: ['resetButton']
+  };
+
+  return composeClasses(slots, getSFSFiltersUtilityClass, classes);
+};
+
+const SFSFiltersRoot = styled('div', {
+  name: 'ESSFSFilters',
+  slot: 'root',
+  overridesResolver: (_, styles) => styles.root
+})(() => ({
+  position: 'relative'
+}));
+
+const SFSFiltersMenuButton = styled(SFSButton, {
+  name: 'ESSFSFilters',
+  slot: 'MenuButton',
+  overridesResolver: (_, styles) => styles.menuButton
+})<{ ownerState: SFSFiltersOwnerState }>(({ theme }) => ({
+  [theme.breakpoints.down('tabletXS')]: {
+    gap: '2px',
+    '& > .MuiTypography-root:first-of-type': {
+      display: 'none'
+    }
+  }
+}));
+
+const SFSDrawer = styled(Drawer, {
+  name: 'ESSFSFilters',
+  slot: 'menu',
+  overridesResolver: (_, styles) => styles.menu
+})(() => ({
+  '.MuiPaper-root': {
+    minWidth: '340px'
+  }
+}));
+
+const SFSFiltersHeader = styled('div', {
+  name: 'ESSFSFilters',
+  slot: 'menuHeader',
```

Comment:
Слоты идут в `PascalCase` и повторяют название подкомпонента. В данном случае будет:
```suggestion
  slot: 'Header',
```